src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -89,14 +89,24 @@ std::string postEntity
     if ((upcrsP->contextElementResponseVector[ix]->statusCode.code != SccOk) &&
         (upcrsP->contextElementResponseVector[ix]->statusCode.code != SccNone))
     {
+      if (upcrsP->contextElementResponseVector[ix]->statusCode.code == SccInvalidParameter)
+      {
+        // Ugly v1 -> v2 error conversion here :(
+        OrionError error(SccRequestEntityTooLarge, "NoResourcesAvailable", "No more than one geo-location attribute allowed");
```

Comment:
This is a little bit strange.
You see a statusCode.code == SccInvalidParameter and directly know that it is a "No more than one geo-location attribute allowed"?
I'm not saying it's wrong, just a little strange and possibly 'dangerous', what if the function that marked SccInvalidParameter is changed and gets more than one reason to flag SccInvalidParameter?
Adding a comment with a NOTE (or even a FIXME), would help.